src/ripple/ledger/impl/View.cpp

```diff
@@ -1355,7 +1355,7 @@ rippleSend (ApplyView& view,
     {
         // VFALCO Why do we need this bCheckIssuer?
         // Direct send: redeeming IOUs and/or sending own IOUs.
-        terResult   = rippleCredit (view, uSenderID, uReceiverID, saAmount, false, j);
+        rippleCredit (view, uSenderID, uReceiverID, saAmount, false, j);
```

Comment:
We don't have to check the return value of `rippleCredit`? That doesn't look right.